cumulusci/tasks/salesforce/install_package_version.py

```diff
@@ -145,30 +165,45 @@ def _run_task(self):
             )
             if "version_number" in self.options:
                 dep.version_number = self.options["version_number"]
-            dep.install(
-                self.project_config,
-                self.org_config,
-                self.install_options,
-                self.retry_options,
-            )
         else:
             dep = PackageNamespaceVersionDependency(
                 namespace=self.options["namespace"],
                 version=version,
                 package_name=self.options["name"],
+                version_id=self.options.get("version_id"),
             )
-            dep.install(
-                self.project_config,
-                self.org_config,
-                self.install_options,
-                self.retry_options,
-            )
+
+        if self.options.get("interactive", False):
+            if dep.version_id:
+                package_desc = self.options["base_package_url_format"].format(
+                    dep.version_id
+                )
+            else:
+                package_desc = str(dep)
+            self.logger.info("Package to install: {}".format(package_desc))
+            if not click.confirm("Continue to install dependencies?", default=True):
```

Comment:
Should we try to validate the `interactive` option to throw an error if it's True but sys.stdin.isatty() is False?

Reply:
Seems reasonable, but I don't know if there are any Windows platform-specific issues with detecting an interactive environment. Any idea?

Reply:
Some quick googling suggests it false-positives on Windows.

Reply:
Oh yeah, glad you thought to check that